### PR TITLE
readme: improve installation + requirements docs

### PR DESCRIPTION
**Pull request by markus-wa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,9 +38,9 @@ Less bugs = better experience = more players for surf!
 * Download and install [Metamod 1.11](https://www.sourcemm.net/downloads.php/?branch=stable)
 * Download and install [SourceMod 1.10](https://www.sourcemod.net/downloads.php?branch=stable) (latest stable) or [SourceMod 1.11](https://www.sourcemod.net/downloads.php?branch=master&all=1) (required for some recommended plugins)
 * Download latest [release](https://github.com/surftimer/SurfTimer/releases/latest) and upload all the files to your csgo server directory
-* Set up A MySQL Database (MySQL 5.7, MySQL 8+, MariaDB supported)
+* Set up A MySQL Database (MySQL 5.7, MySQL 8+, MariaDB supported) and [initialise a schema for surftimer](https://github.com/surftimer/SurfTimer/blob/master/scripts/mysql-files/fresh_install.sql)
 * Add a MySQL database called surftimer to csgo/addons/sourcemod/configs/databases.cfg
-* Ensure [End-Touch-Fix](https://github.com/rumourA/End-Touch-Fix) is loaded, this is required to ensure times are always accurate
+* Ensure [End-Touch-Fix](https://github.com/rumourA/End-Touch-Fix) is loaded, this is required to ensure times are always accurate (note: this should be included and loaded by SurfTimer automatically without installing separately)
```
Comment (Bara) [inline README.md:43]: EndTouchFix is not included in Surftimer, it's included inside the build package and will loaded, which is it, as standalone Plugin. Note should something like "included inside the build package" or so.

Comment (markus-wa) [inline README.md:43]: thanks!

```suggestion
* Ensure [End-Touch-Fix](https://github.com/rumourA/End-Touch-Fix) is loaded, this is required to ensure times are always accurate (note: this is included in the official SurfTimer release build and loaded automatically without installing separately unless you are building SurfTimer yourself)
```

Comment (markus-wa) [reply README.md]: updated it - let me know if this is better :slightly_smiling_face:

File: README.md [hunk 2]
```diff
@@ -38,9 +38,9 @@ Less bugs = better experience = more players for surf!
...
 * Download and install [SourceMod 1.10](https://www.sourcemod.net/downloads.php?branch=stable) (latest stable) or [SourceMod 1.11](https://www.sourcemod.net/downloads.php?branch=master&all=1) (required for some recommended plugins)
 * Download latest [release](https://github.com/surftimer/SurfTimer/releases/latest) and upload all the files to your csgo server directory
-* Set up A MySQL Database (MySQL 5.7, MySQL 8+, MariaDB supported)
+* Set up A MySQL Database (MySQL 5.7, MySQL 8+, MariaDB supported) and [initialise a schema for surftimer](https://github.com/surftimer/SurfTimer/blob/master/scripts/mysql-files/fresh_install.sql)
 * Add a MySQL database called surftimer to csgo/addons/sourcemod/configs/databases.cfg
-* Ensure [End-Touch-Fix](https://github.com/rumourA/End-Touch-Fix) is loaded, this is required to ensure times are always accurate
+* Ensure [End-Touch-Fix](https://github.com/rumourA/End-Touch-Fix) is loaded, this is required to ensure times are always accurate (note: this is included in the official SurfTimer release build and loaded automatically without installing separately unless you are building SurfTimer yourself)
```
Comment (markus-wa) [inline README.md:43]:
```suggestion
* Ensure [End-Touch-Fix](https://github.com/rumourA/End-Touch-Fix) is loaded, this is required to ensure times are always accurate (note: this is included in the official SurfTimer release build and loaded automatically without installing separately unless you are compiling SurfTimer yourself)
```

**Comment by sneak-it** [comment]
The whole requirements section needs to be revamped instead of being band-aided, because even with the PR'd changes the entire section is misleading. Suggested/recommended/optional plugins and extensions are not any sort of requirement, thus should not be under the requirements header. Breaking Requirements out out into 'Runtime Dependencies', 'Compilation Requirements', and 'Suggested Plugins/Extensions' would make things definitively clear.

**Review by dPexxIAM — APPROVED** [review]

**Pull request merged**